### ISSUE TITLE
anticipation of problem with species names in R

### ISSUE THREAD

**Issue by tmesaglio** [tmesaglio/duck-river-traits#1]
R doesn't like hybrid symbols

how do I deal with entities that are 'Genus sp.'? (keep the sp. or keep as just the genus name?)

how to match my Freesia with Price's Freesia (different names but essentially same thing)

**Comment by tmesaglio** [comment]
just adding onto the hybrid symbol issue;

if a cell contains a 'dodgy' character in R, all contents of that cell become completely unsearchable.
You can test this with the file 'species_lists', row 723, column 1, which contains Crocosmia x crocosmiiflora. If you search for any part of that name, you get zero results. Same for the other hybrid taxa

**Comment by wcornwell** [comment]
yup, got to decide what to do on this. the hybrid symbol is not in UTF-8 which creates issues going back and forth between different software.  Maybe change them all to x which will then match APC?

**Comment by wcornwell** [comment]
<img width="1605" alt="Screenshot 2022-12-14 at 11 27 14 am" src="https://user-images.githubusercontent.com/5534161/207474414-c69fb525-0683-4854-875a-070158f53cc0.png">

**Comment by wcornwell** [comment]
this 
```
sl<-read_csv("species_lists.csv")
sl$species_f<-iconv(sl$species, "UTF-8", "UTF-8",sub='x') 
```
seems to fix it

**Comment by wcornwell** [comment]
<img width="1072" alt="Screenshot 2022-12-14 at 11 31 46 am" src="https://user-images.githubusercontent.com/5534161/207475037-edce6061-fe84-45f6-8821-f4fe581dd459.png">
